iati/core/schemas.py

```diff
@@ -52,25 +52,19 @@ def __init__(self, name=None):
             Create test instance where the SchemaError is raised.
 
         """
-        self.name = name
         self._schema_base_tree = None
         self.codelists = set()
         self.rulesets = set()
+        self.source_path = path
```

Comment:
Unclear on what its use is without more context, but may want to be a private attribute.